portal-frontend/test/tests/submissions/TUR_submission_creation.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { test } from '@playwright/test';
+
+const baseUrl = 'http://localhost:4201/login';
```

Comment:
Since these are copied between files, could we use a central config file?

Reply:
I wasn't thinking of config library, could you just make a config.ts in the same folder that these import from?

Reply:
done